src/test/java/driver/ChromeSingleton.java

```diff
@@ -4,23 +4,14 @@
 import org.openqa.selenium.chrome.ChromeOptions;
 
 public class ChromeSingleton {
-    private static ChromeSingleton instance;
-    private static ChromeDriver chromeDriver;
+    private static ChromeDriver instance;
 
-    private ChromeSingleton(){
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--start-maximized");
-        chromeDriver = new ChromeDriver(options);
-    }
-    private static synchronized ChromeSingleton getInstance(){
+    public static synchronized ChromeDriver getInstance(){
```

Comment:
this impl is much better than the previous one, but
1. synchronized method - is not the best option
2. also, think about one singleton that will manage both(all) drivers

Reply:
im talking about "merging" your 2 singletons and driver factory